automation/taskcluster/lib/tasks.py

```diff
@@ -60,11 +60,12 @@ def craft_assemble_release_task(self, architectures, channel, is_staging, versio
             'python automation/taskcluster/helper/get-secret.py -s {} -k {} -f {}'.format(
                 secret_index, key, target_file
             )
-            for key, target_file in (
+            for key, target_file in filter(None.__ne__, (
```

Comment:
Is this readable enough? It's removing all `None`s from the list.
I could make this a list comprehension instead :thinking: